public/css/breach-stats.css

```diff
@@ -19,19 +18,13 @@
 }
 
 .breach-stat-number {
-  font-size: 50px;
+  font-size: 56px;
+  line-height: 1.14;
   min-width: 2rem;
```

Comment:
Why the `rem` unit on width here, and `px` sizing for font-size. I would recommend either flipping these values or making the same. This would give them the same scalability/precision.

Reply:
Probably just an old holdover from The Time Before We Started Using Pixels Everywhere. I'll update.